src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -150,6 +150,8 @@ void RendererOpenGL::SwapBuffers() {
     {
         auto aggregator = Common::Profiling::GetTimingResultsAggregator();
         aggregator->AddFrame(profiler.GetPreviousFrameResults());
+        if (VideoCore::g_toggle_framelimit_enabled)
+        FrameLimiter(aggregator->GetAggregatedResults().frame_time.avg, 60);
```

Comment:
As hardware timestamps are also calculated in ticks I think it's perfectly reasonable to use us or ms as unit as oposed to seconds in a float